README.md

```diff
@@ -146,23 +146,39 @@ see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod)
 
 ## Get in touch
 
-+ cockroach-db@googlegroups.com
-+ \#cockroachdb on freenode
+We spend almost all of our time here on GitHub, and use the [issue
+tracker](https://github.com/cockroachdb/cockroach/issues) for
+bug reports and development-related questions.
 
-## Contributing
+For anything else, message our mailing list cockroach-db@googlegroups.com.
+
+### Contributing
+
+We're an Open Source project and welcome contributions.
+See [CONTRIBUTING.md](https://github.com/cockroachdb/cockroach/blob/master/CONTRIBUTING.md) to get your local environment set up.
+Once that's done, participating in [code reviews](https://github.com/cockroachdb/cockroach/pulls/) is the best way to get started and to find an interesting problem to work on.
+
+## Talks
+
```

Comment:
I think you could also list:

* [Venue: Data Driven NYC]
 (https://youtu.be/TA-Jw78Ms_4), by [Spencer Kimball] (https://github.com/spencerkimball) on (06/16/2015), 23min.<br />
   A shorter, less technical presentation of Cockroach.